system/authorized_key.py

```diff
@@ -460,6 +475,7 @@ def main():
            key_options = dict(required=False, type='str'),
            unique      = dict(default=False, type='bool'),
            exclusive   = dict(default=False, type='bool'),
+	   validate_certs = dict(default=True, type='bool'),
```

Comment:
It's tabs. Line 126 and 478 have tabs rather than spaces. Please check your vim settings for softtabstop and expandtab.